README.md

```diff
@@ -78,7 +78,7 @@ In Eclipse, create a Debug Configuration by selecting the cgeo application (insi
 
 The Test classes can be found in the project test. Test classes should be located in the same package as
 the class under test.
-Every class can be "Run As" (or "Debug As") an [Android JUnit Test](http://developer.android.com/guide/topics/testing/testing_android.html) from Eclipse.
+Every class can be "Run As" (or "Debug As") an [Android JUnit Test](https://developer.android.com/training/testing/fundamentals.html).
```

Comment:
Did you removed the ` from Eclipse` on purpose?

Reply:
I don't mind if Eclipse is removed here. The default description should target Android Studio now.
But the menu entry in Android Studio is not called `Run As`, it's `Run '<class name>'`.